www/source/partials/docs/_using-hab-config-workstation.html.md.erb

```diff
@@ -5,7 +5,7 @@ Once Habitat has been installed, the <code>hab</code> CLI makes it easy to get y
 
 Setup asks you to create a new origin and a set of origin keys.
 
-Optionally, you can also provide a Habitat personal access token to upload packages to the public depot and share them with the Habitat community. See the [access token documentation](/docs/using-builder/#builder-token) for details on generating and using your access token.
+Optionally, you can also provide a Habitat personal access token to upload packages to the public depot and share them with the Habitat community. See the [access token documentation](/docs/using-builder/#builder-token) for details on generating and using your access token. You can also add a previously-generated control gateway secret that is used to authenticate commands sent to a remote Supervisor. See the [control gateway](/docs/internals#control-gateway) documentation for more information.
```

Comment:
I don't think this really belongs here and can be left out. See https://www.habitat.sh/docs/install-habitat/#configure-workstation